bugbug/model.py

```diff
@@ -193,18 +193,40 @@ def train(self, importance_cutoff=0.15):
             shap_values = explainer.shap_values(X_train)
 
             # TODO: Actually implement feature importance visualization for multiclass problems.
+            new_shap_values = 0
             if isinstance(shap_values, list):
-                shap_values = np.sum(np.abs(shap_values), axis=0)
+                new_shap_values = np.sum(np.abs(shap_values), axis=0)
+            else:
+                new_shap_values = shap_values
 
             important_features = self.get_important_features(
-                importance_cutoff, shap_values
+                importance_cutoff, new_shap_values
             )
 
-            print(f"\nTop {len(important_features)} Features:")
-            for i, [importance, index, is_positive] in enumerate(important_features):
-                print(
-                    f'{i + 1}. \'{feature_names[int(index)]}\' ({"+" if (is_positive) else "-"}{importance})'
-                )
+            if not isinstance(shap_values, list):
+                print(f"\nTop {len(important_features)} Features:")
+                for i, [importance, index, is_positive] in enumerate(
+                    important_features
+                ):
+                    print(
+                        f'{i + 1}. \'{feature_names[int(index)]}\' ({"+" if (is_positive) else "-"}{importance})'
+                    )
+            else:
+                print("{:<}".format("classes"), end="   ")
+                for importance, index, is_pos in important_features:
+                    print("{:>}".format(feature_names[int(index)]), end="  ")
+                print("\n")
+                for class_num, item in enumerate(shap_values):
+                    print("class {}".format(class_num), end="   ")
```

Comment:
We should print the class name rather than the class number

Reply:
I've assumed the classes be in ascending order of their names, i don't know if it is correct :p

Reply:
@marco-c this should be ascending or descending? cause you have used descending order in summary plot

Reply:
Not sure, we need to confirm with a real case and see if the results make sense in ascending or in descending order.